manifests/j/Jidifudi/GT4T/8.49.250310/Jidifudi.GT4T.installer.yaml

```
@@ -0,0 +1,30 @@
+# Automatically updated by the winget bot at 2025/Mar/10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Jidifudi.GT4T
+PackageVersion: 8.49.250310
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: 3E06249EDEA6318A70E582B27B7D8C58F59ABA83708E686BDE415FE0CBE57804
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- Architecture: x64
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_64_bit/GT4T_setup.exe
+  InstallerSha256: 7DD9882C71FC0E245EE3071BD2CCAE83158211A75399B9CB03C8270FA789B885
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerUrl: https://dl.gt4t.cn/downloads/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: 3E06249EDEA6318A70E582B27B7D8C58F59ABA83708E686BDE415FE0CBE57804
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- InstallerLocale: zh-CN
+  Architecture: x64
+  InstallerUrl: https://dl.gt4t.cn/downloads/apps/gt4t_win_64_bit/GT4T_setup.exe
+  InstallerSha256: 7DD9882C71FC0E245EE3071BD2CCAE83158211A75399B9CB03C8270FA789B885

```

```suggestion
  InstallerSha256: 7727875514A24551101983F402C64A13D16EC77F91FD61E8CAB3BA86DC04661A
```

(Automated response - build 1007.)